### ISSUE TITLE
New folder for resources?

### ISSUE THREAD

**Issue by starktech23** [HuangJunye/Qiskit-Hackathon-Guide#3]
Add general resources for hackathon stuff that doesn't come under the existing categories.

Example content: https://medium.com/qiskit/6-tips-for-an-amazing-qiskit-hackathon-presentation-e6cea20ce3b3

**Comment by HuangJunye** [comment]
Good idea. Go ahead and add them please! Maybe you should change it first in the develop branch and do a PR

**Comment by starktech23** [comment]
Sure thing, I'll work on a format and add them.
